src/fuzz/meson.build

```diff
@@ -14,3 +13,8 @@ simple_fuzzers += files(
         'fuzz-varlink.c',
         'fuzz-varlink-idl.c',
 )
+
+# The following fuzzers do not work on oss-fuzz. See #11018.
+if not want_ossfuzz
+        simple_fuzzers += files('fuzz-compress.c')
+endif
```

Comment:
I'd add my favourite quote here to make it clear that it's done deliberately
>  just because google's unhealthy fixation with static libraries seeps into everything they do, including that particular testing infrastructure, and allow them to set the lowest common denominator

Reply:
I do not think it is necessary to complain an other project here. Linking to #11018 should be enough.

Reply:
It's not about complaining. It's about the sort of arguments people are going to get if they try to make that stuff work upstream (which is a waste of time). It's actually possible to get that fuzz target (and a bunch of other fuzz targets that don't exist upstream) to work but some things are still broken of course: https://github.com/google/oss-fuzz/issues/12959